spring-ai-spring-boot-starters/spring-ai-starter-vector-store-milvus/pom.xml

```diff
@@ -48,6 +48,12 @@
             <version>${project.parent.version}</version>
         </dependency>
 
```

Comment:
The dependency `spring-ai-spring-boot-autoconfigure` needs to be removed. I will fix when merging.